docs/marketplace/dev/feature-integration-guides/202200.0/product-approval-process-feature-integration.md

```diff
@@ -0,0 +1,740 @@
+---
+title: Product Approval Process feature integration
```

Comment:
please mention changes for `data/import/common/common/glossary.csv` for this guide